src/test/scala/com/mwz/sonar/scala/scapegoat/ScapegoatInspectionsSpec.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Sonar Scala Plugin
+ * Copyright (C) 2018 All contributors
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+package com.mwz.sonar.scala.scapegoat
+
+import org.scalatest.{FlatSpec, Inspectors, Matchers}
+import org.sonar.api.batch.rule.Severity
+
+/** Tests the generated scapegoat inspections file */
+class ScapegoatInspectionsSpec extends FlatSpec with Inspectors with Matchers {
+  "The Scapegoat Inspections object" should "define all scapegoat inspections" in {
+    ScapegoatInspections.AllInspections should have size 126
+    ScapegoatInspections.AllInspections.distinct should have size 126
```

Comment:
Is your concern here that the inspections might be duplicates because of our logic or what comes from Scapegoat?

Reply:
To be honest, both. I just want to be sure each inspection is unique.